carbonserver/render.go

```diff
@@ -711,9 +708,9 @@ func (listener *CarbonserverListener) Render(req *protov2.MultiFetchRequest, str
 	valuesFetched := 0
 	fetchSize := 0
 	// TODO: should chan buffer size be configurable?
-	responseChan := make(chan response, 1000)
+	streamChan := make(chan response, 10000)
```

Comment:
This variable is is used quite a lot and quite far from definition, so I think it's important for it to have intuitive name. streamChan naming doesn't give any intuition on what's inside, maybe something like `preparedResponsesChan` would be more clear.

Reply:
Changed it to `responseChanToStream`. Because it is the channel which contains the responses which will be streamed.

Reply:
BTW, They are not marshalled.

Reply:
ah indeed not marshalled, that was for http, cool thx